challenges/objects-arrays.js

```diff
@@ -97,23 +135,24 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 
 */
 
-const lowerCase = [];
+const lowerCase = zooAnimals.map(item => item.animal_name.toLowerCase());
 console.log(lowerCase); 
+console.log(typeof lowerCase)
 
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const lowerPopulation = [];
+const lowerPopulation = zooAnimals.filter(item => item.population <= 5);
 console.log(lowerPopulation);
 
 /* Request 4: .reduce() 
 
 The zoos need to know their total animal population across the United States.  Find the total population from all the zoos using the .reduce() method.
 
 */
-const populationTotal = 0;
+const populationTotal = zooAnimals.reduce((plus, item) => plus + item.population, 0);
```

Comment:
Nice! Again, I'd suggest being more specific with your naming in methods ie use population instead of item. Remember you're writing code for the next dev as well.